### PR TITLE
Fetch all agents configurations in //

### PR DESCRIPTION
**Pull request by lasryaric**

### CODE REVIEW

File: front/lib/api/assistant/global_agents.ts [hunk 1]
```diff
@@ -423,34 +423,25 @@ export async function getGlobalAgents(
...
-    globalAgents.push(notionAgent);
-  }
-  const githubAgent = await _getGithubGlobalAgent(auth);
-  if (githubAgent) {
-    globalAgents.push(githubAgent);
-  }
-  const dustAgent = await _getDustGlobalAgent(auth);
-  if (dustAgent) {
-    globalAgents.push(dustAgent);
+  const agentCandidates = await Promise.all([
```
Comment (philipperolet) [inline front/lib/api/assistant/global_agents.ts:426]: I would do it with
```
await Promise.all(GLOBAL_AGENTS_SID.values().map((a) => getGlobalAgent(auth, a.sId)))
```
The reason is, if we add or remove a global agent, it's easy to forget to update the list here 
Better to have one single source of truth for the global agent list

As a nice to have it is more concise

**Review by philipperolet — CHANGES_REQUESTED** [review]
Little change, otherwise good!

File: front/lib/api/assistant/global_agents.ts (hunk 1 above)
Comment (spolu) [reply front/lib/api/assistant/global_agents.ts:426]: Now that ordering is supposed to be handled client-side I guess it's a fair ask?

File: front/lib/api/assistant/global_agents.ts [hunk 2]
```diff
@@ -423,34 +423,25 @@ export async function getGlobalAgents(
...
+    _getClaudeGlobalAgent(),
+    _getClaudeInstantGlobalAgent(),
+    _getSlackGlobalAgent(auth),
+    _getGoogleDriveGlobalAgent(auth),
+    _getNotionGlobalAgent(auth),
+    _getGithubGlobalAgent(auth),
+    _getDustGlobalAgent(auth),
+  ]);
+
+  const globalAgents: AgentConfigurationType[] = [];
```
Comment (spolu) [inline front/lib/api/assistant/global_agents.ts:439]: return agentCandidates.filter() ?

File: front/lib/api/assistant/global_agents.ts (hunk 1 above)
Comment (lasryaric) [reply front/lib/api/assistant/global_agents.ts]: @philipperolet agree now. Thanks for catching it again!

**Review by spolu — APPROVED** [review]

**Review by philipperolet — APPROVED** [review]
LGTM :+1:

**Pull request merged**